AAI/AAIConnectProfile.md

```diff
@@ -400,29 +439,64 @@ the Broker.
     except for accommodating for `exp` timestamps to be represented as
     indicated above.
 
-#### Conformance for Claim Clearinghouses (consuming Access Tokens to give access to data)
+#### Conformance for Passport Issuers
 
-1.  Claim Clearinghouses MUST trust at least one Broker.
+TODO "GA4GH Claims Tokens" perhaps should be "Visas"
 
-    1.  Claim Clearinghouses MAY trust more than one Broker
+1.  Passport Issuers are used to re-package and re-sign GA4GH Claims Tokens, nominally JWTs, that contain Visas.
```

Comment:
We don't need to block this PR on this, but I think this section could benefit from a diagram similar to the "Flow of Claims" section that illustrates an example where the Broker is also a Passport Issuer.

Reply:
"GA4GH Claims Tokens" are not defined, it should be "Passports"